.golangci.yml

```diff
@@ -0,0 +1,22 @@
+linters:
+  disable-all: true
+  enable:
+    - govet
+    - errcheck
```

Comment:
I'm not sure about this one.  I think I'd rather have it off than have the extra `_ =` assignments and `//nolint:errcheck` comments.

Reply:
Done